nemo_curator/utils/script_utils.py

```diff
@@ -556,10 +573,12 @@ def parse_semdedup_args(
         argumentHelper = ArgumentHelper(parser)
         argumentHelper.add_distributed_args()
         if add_input_args:
-            argumentHelper.add_arg_input_data_dir(required=True)
+            argumentHelper.add_arg_input_data_dir()
```

Comment:
Why did we drop `required=True` here ?

Reply:
So for our CLI scripts, the input data directory is only needed for step 1 of 3, but some of the other input arguments (ID field, ID datatype) are used in subsequent steps. I figured since `parse_semdedup_args` is used for all 3 steps, removing the requirement is okay.